### PR TITLE
S101.07_also_send_api_base_url_with_token

### PR DESCRIPTION
**Pull request by kulltc**

### CODE REVIEW

File: Lib/Salesforce/SalesforceAuthManager.php [hunk 1]
```diff
@@ -92,11 +95,14 @@ public function getAuthContainer($userId) {
 		$salesforce = $this->salesforce;
```
Comment (expand-dev) [inline Lib/Salesforce/SalesforceAuthManager.php]: MAJOR Rename "$salesforce" which has the same name as the field declared at line 17. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS1117)

**Comment by expand-dev** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 1 minor

Watch the comments in this conversation to review them.

#### 3 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SalesforceAuthManager.php](https://github.com/ExpandOnline/AuthManager/blob/5e44c8a57b1bf72ec445f480fac96e147cee3451/Lib/Salesforce/SalesforceAuthManager.php): Refactor this file to either declare symbols or cause side effects, but not both. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2036)
1. ![MAJOR][MAJOR] [SalesforceAuthManager.php#L25](https://github.com/ExpandOnline/AuthManager/blob/5e44c8a57b1bf72ec445f480fac96e147cee3451/Lib/Salesforce/SalesforceAuthManager.php#L25): Remove this creation of object in constructor. Use dependency injection instead. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2830)
1. ![MINOR][MINOR] [SalesforceAuthManager.php](https://github.com/ExpandOnline/AuthManager/blob/5e44c8a57b1bf72ec445f480fac96e147cee3451/Lib/Salesforce/SalesforceAuthManager.php): Replace all tab characters in this file by sequences of white-spaces. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS105)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: Lib/Salesforce/SalesforceAuthManager.php [hunk 2]
```diff
@@ -27,6 +27,9 @@ public function __construct() {
 			'clientSecret'      => Configure::read('Salesforce.client_secret'),
 			'redirectUri'       => $this->_getCallbackUrl(MediaPlatform::SALESFORCE)
 		]);
+		if (Configure::read('DEO.live') !== true) {
+			$this->salesforce->setDomain('https://test.salesforce.com');
```
Comment (ronoudgenoeg) [inline Lib/Salesforce/SalesforceAuthManager.php]: De salesforce url (of environment) moet een configuratie zijn in authmanager, niet configuratie van DEO gebruiken.

**Comment by expand-dev** [comment]
SonarQube analysis reported 5 issues
* ![MAJOR][MAJOR] 3 major
* ![MINOR][MINOR] 2 minor

Watch the comments in this conversation to review them.

#### 4 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [SalesforceAuthManager.php](https://github.com/ExpandOnline/AuthManager/blob/4481e50cd44ee4ee9e3b771a4b9457badcf9ab68/Lib/Salesforce/SalesforceAuthManager.php): Refactor this file to either declare symbols or cause side effects, but not both. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2036)
1. ![MAJOR][MAJOR] [SalesforceAuthManager.php#L25](https://github.com/ExpandOnline/AuthManager/blob/4481e50cd44ee4ee9e3b771a4b9457badcf9ab68/Lib/Salesforce/SalesforceAuthManager.php#L25): Remove this creation of object in constructor. Use dependency injection instead. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS2830)
1. ![MINOR][MINOR] [bootstrap.php](https://github.com/ExpandOnline/AuthManager/blob/4481e50cd44ee4ee9e3b771a4b9457badcf9ab68/Config/bootstrap.php): Replace all tab characters in this file by sequences of white-spaces. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS105)
1. ![MINOR][MINOR] [SalesforceAuthManager.php](https://github.com/ExpandOnline/AuthManager/blob/4481e50cd44ee4ee9e3b771a4b9457badcf9ab68/Lib/Salesforce/SalesforceAuthManager.php): Replace all tab characters in this file by sequences of white-spaces. [rule](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS105)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by kulltc** [comment]
@ronoudgenoeg Done!

**Pull request merged**